src/bindeps_conda.jl

```diff
@@ -1,57 +1,46 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
 using BinDeps
 
-type Manager <: BinDeps.PackageManager
+type ManagerType{T} <: BinDeps.PackageManager
```

Comment:
`ManagerType` is confusing, since both `Manager` and `ManagerType` are types.  Maybe rename `ManagerType` to `EnvManager`?